phoneUtil/src/main/java/pl/washboard/hvac/libutil/PhoneFormater.java

```diff
@@ -0,0 +1,36 @@
+package pl.washboard.hvac.libutil;
+
+import com.google.i18n.phonenumbers.NumberParseException;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import com.google.i18n.phonenumbers.Phonenumber;
+
+public class PhoneFormater {
+    private static final PhoneNumberUtil PHONE_NUMBER_UTIL = PhoneNumberUtil.getInstance();
+
+    public String national(String str) {
+        try {
+            Phonenumber.PhoneNumber NumberProto = PHONE_NUMBER_UTIL.parse(str, "PL");
+            return PHONE_NUMBER_UTIL.format(NumberProto, PhoneNumberUtil.PhoneNumberFormat.NATIONAL);
+        } catch (NumberParseException e) {
+            return str;
+        }
+    }
+
+    public String international(String str) {
+        try {
+            Phonenumber.PhoneNumber NumberProto = PHONE_NUMBER_UTIL.parse(str, "PL");
+            return PHONE_NUMBER_UTIL.format(NumberProto, PhoneNumberUtil.PhoneNumberFormat.INTERNATIONAL);
+        } catch (NumberParseException e) {
+            return str;
+        }
+    }
+
+    public PhoneNumberUtil.PhoneNumberType getType(String str) {
+        try {
+            Phonenumber.PhoneNumber NumberProto = PHONE_NUMBER_UTIL.parse(str, "PL");
+            return PHONE_NUMBER_UTIL.getNumberType(NumberProto);
+        } catch (NumberParseException e) {
+            return null;
```

Comment:
jak zwracasz cokolowiek z try/catch to z reguly log nalezy zrobic
log.error a jak tam nie mazs loggera niby to System.err.prinltn('opiusjesz ocb...
i zwracasz chociazby null'a

druga opcja to rzucasz runtimeexception swoj czy tam nawet ten sam nie lapiesz go, ale warto czasem zaznavzyc ze jezeli metoda nie bedzie w stanie ogarnac co to za typ bo zle argumenty to rzuci blad i w metodzie to wywulujacej robisz try/ctch i lapiez i tam np zwracasz 'nie wiem co to za typ bo polecial blad bo z dupy dane'
albo robisz if(..==null) nie wiem co to za typ bo polecial...

zalezy co chcesz osiagnac, obsluga sytuacji wyjatkowych jest wzglednie kosztowna z punktu widzenia operacji do wykonania bo trrzbea zebrc stacka, zapisac kontekst itp zlozyc stos wywolan itp itd. wiec lapanie i rzucanie jak jet blad rzedzywsicie, a nie zawsze, 

czyli tu zalezy od tego co chcesz zeby funkcja robila, jak nie chcez zeby rzcuala bledem to null , jak nie chcesz nulla to mozesz nie lapac bledu i np. dopisac throws NnumberParseExceptin zeby ostrzec progeramiste ze moze takie cos byc